doc/classes/String.xml

```diff
@@ -549,78 +617,79 @@
 			</description>
 		</method>
 		<method name="replace" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="what" type="String" />
 			<param index="1" name="forwhat" type="String" />
 			<description>
-				Replaces occurrences of a case-sensitive substring with the given one inside the string.
+				Replaces all occurrences of [param what] inside the string with the given [param forwhat].
 			</description>
 		</method>
 		<method name="replacen" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="what" type="String" />
 			<param index="1" name="forwhat" type="String" />
 			<description>
-				Replaces occurrences of a case-insensitive substring with the given one inside the string.
+				Replaces all [b]case-insensitive[/b] occurrences of [param what] inside the string with the given [param forwhat].
 			</description>
 		</method>
 		<method name="rfind" qualifiers="const">
 			<return type="int" />
 			<param index="0" name="what" type="String" />
 			<param index="1" name="from" type="int" default="-1" />
 			<description>
-				Returns the index of the [b]last[/b] case-sensitive occurrence of the specified string in this instance, or [code]-1[/code]. Optionally, the starting search index can be specified, continuing to the beginning of the string.
+				Returns the index of the [b]last[/b] occurrence of [param what] in this string, or [code]-1[/code] if there are none. The starting search index can be specified with [param from], continuing to the beginning of the string. This method is the reverse of [method find].
 			</description>
 		</method>
 		<method name="rfindn" qualifiers="const">
 			<return type="int" />
 			<param index="0" name="what" type="String" />
 			<param index="1" name="from" type="int" default="-1" />
 			<description>
-				Returns the index of the [b]last[/b] case-insensitive occurrence of the specified string in this instance, or [code]-1[/code]. Optionally, the starting search index can be specified, continuing to the beginning of the string.
+				Returns the index of the [b]last[/b] [b]case-insensitive[/b] occurrence of [param what] in this string, or [code]-1[/code] if there are none. The starting search index can be specified with [param from], continuing to the beginning of the string. This method is the reverse of [method findn].
 			</description>
 		</method>
 		<method name="right" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="length" type="int" />
 			<description>
-				Returns a number of characters from the right of the string. If negative [param length] is used, the characters are counted downwards from [String]'s length.
-				[b]Example:[/b]
+				Returns the last [param length] characters from the end of the string. If [param length] is negative, strips the first [param length] characters from the string.
```

Comment:
Would suggest like earlier harmonizing the two sentences by adding "the beginning of the string" in the 2nd one.